### PR TITLE
Translate

### PR DESCRIPTION
**Pull request by akosd127**

### CODE REVIEW

File: docs/translations/hu/scripting/funcitons/SetVehicleAngularVelocity.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+title: SetVehicleAngularVelocity
+description: Beállítja a jármű X, Y és Z szögsebességét.
+tags: ["vehicle"]
+---
+
+<VersionWarn version='SA-MP 0.3b' />
+
+:::info
+
+Ez a függvény a _world_ térben van, nem a _local_ térben. Ha a helyi tér szögsebességét szeretné módosítani, akkor a [jármű rotációs quat] (GetVehicleRotationQuat) alapján elforgatási mátrixot kell alkalmaznia.
```
Comment (Southclaws) [inline docs/translations/hu/scripting/funcitons/SetVehicleAngularVelocity.md:11]:
```suggestion
Ez a függvény a _world_ térben van, nem a _local_ térben. Ha a helyi tér szögsebességét szeretné módosítani, akkor a [jármű rotációs quat](GetVehicleRotationQuat) alapján elforgatási mátrixot kell alkalmaznia.
```

**Review by Southclaws — CHANGES_REQUESTED** [review]
Looks good except this minor issue with the hyperlink and the file path. There's a slight typo in the path name, the word `functions` is spelt `funcitons` which will prevent this page from appearing in the sidebar:

```diff
-docs/translations/hu/scripting/funcitons/SetVehicleAngularVelocity.md
+docs/translations/hu/scripting/functions/SetVehicleAngularVelocity.md
```

**Review by akosd127 — COMMENTED** [review]
.

**Review by leonardssh — APPROVED** [review]

**Review by akosd127 — COMMENTED** [review]
solved

**Review by adib-yg — APPROVED** [review]

**Pull request merged**